tinygrad/tensor.py

```diff
@@ -222,8 +222,6 @@ def tolist(self) -> Union[Sequence[ConstType], ConstType]:
     """
     Returns the value of this tensor as a nested list.
 
-    Currently this only works for flattened tensors.
-
```

Comment:
does supporting nested lists mean tensors that aren't flat also work?
<img width="250" alt="image" src="https://github.com/tinygrad/tinygrad/assets/135171913/3e1c7421-1693-4ab9-aaf3-a78c1155352c">